src/plugins/data/public/search/expressions/esaggs.ts

```diff
@@ -248,7 +252,7 @@ export const esaggs = (): EsaggsExpressionFunctionDefinition => ({
       multi: true,
     },
   },
-  async fn(input, args, { inspectorAdapters, abortSignal }) {
+  async fn(input, args, { inspectorAdapters, abortSignal, search }) {
```

Comment:
Does it make sense to add this as part of `args` instead of `handlers`? Or is it even feasible?

I guess I'm not even sure how this is being passed in... Was `search` being passed in prior to this PR inside the third argument here?

Reply:
it will need to be a handler, i wouldn't include it as part of initial input however to make it more explicit and would prefer a separate getter function on the handlers